addons/web/controllers/database.py

```diff
@@ -124,7 +124,7 @@ def drop(self, master_pwd, name):
             return self._render_template(error=error)
 
     @http.route('/web/database/backup', type='http', auth="none", methods=['POST'], csrf=False)
-    def backup(self, master_pwd, name, backup_format='zip'):
+    def backup(self, master_pwd, name, backup_format='zip', no_filestore=False):
```

Comment:
don't use double-negative
```suggestion
    def backup(self, master_pwd, name, backup_format='zip', filestore=True):
```